app/lib/konnectors.js

```diff
@@ -89,7 +89,7 @@ export function run (cozy, slug, accountId, folderId, timeout = 120 * 1000) {
       }
     })
   })
-  .then(job => waitForJobFinished(cozy, job, timeout))
+  //.then(job => waitForJobFinished(cozy, job, timeout))
```

Comment:
Because, the stack does not offer a way to get a running job at the moment. But the back is working on it.